conf.pri.mingw

```diff
@@ -0,0 +1,43 @@
+# Adapt these paths
+# Point these to the include folders
+INCLUDEPATH += $$quote(C:/qBittorrent/boost_1_51_0)
+INCLUDEPATH += $$quote(C:/qBittorrent/RC_0_16/include)
+INCLUDEPATH += $$quote(C:/qBittorrent/Zlib/include)
+INCLUDEPATH += $$quote(C:/qBittorrent/openssl/include)
+# Point these to the lib folders
+LIBS += $$quote(-LC:/qBittorrent/boost_1_51_0/stage/lib)
+LIBS += $$quote(-LC:/qBittorrent/RC_0_16/bin/path-according-to-the-build-options-chosen)
+LIBS += $$quote(-LC:/qBittorrent/Zlib/lib)
+LIBS += $$quote(-LC:/qBittorrent/openssl/lib)
+
+# Adapt the lib names/versions accordingly
+CONFIG(debug, debug|release) {
+    LIBS += libtorrent-rasterbar \
+            libboost_system-mt \
+            libboost_filesystem-mt \
+            libboost_thread_win32-mt
+}
+else {
+    LIBS += libtorrent-rasterbar \
+            libboost_system-mt \
+            libboost_filesystem-mt \
+            libboost_thread_win32-mt
+}
+
+LIBS += libadvapi32 libshell32 libuser32 libole32
+LIBS += libcrypto libssl libwsock32 libws2_32 libz libiconv
+LIBS += libpowrprof
+
+DEFINES += NTDDI_VERSION=0x05010000
+DEFINES += _WIN32_WINNT=0x0501
+DEFINES += _WIN32_IE=0x0501
+
+DEFINES += BOOST_ALL_NO_LIB
+# Boost 1.60+ defaults to Vista+ support. The define below enables XP support again.
+DEFINES += BOOST_USE_WINAPI_VERSION=0x0501
+# Disable this if building against libtorrent 1.1.x (RC_1_1)
+DEFINES += BOOST_ASIO_SEPARATE_COMPILATION
+DEFINES += BOOST_SYSTEM_STATIC_LINK=1
```

Comment:
This controls dynamic linking. Please, move it to the boost LIBS section and add a comment explaining that either explicit library names can be given together with `BOOST_ALL_NO_LIB` definition, or library names can be controlled via this definition.

Reply:
I think we can omit this definition since it is default: http://www.boost.org/doc/libs/1_65_1/boost/system/config.hpp